## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 513782`
Commit: [`1f39597a2c8bec6b140e8db6b90a9817ab641f92`](https://github.com/NixOS/nixpkgs/commit/1f39597a2c8bec6b140e8db6b90a9817ab641f92) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/1f39597a2c8bec6b140e8db6b90a9817ab641f92..pull/513782/head))
Merge: [`a2ac83a10fe90f3835f0da07bbb8784366d0fae3`](https://github.com/NixOS/nixpkgs/commit/a2ac83a10fe90f3835f0da07bbb8784366d0fae3)

Logs: https://github.com/felbinger/nixpkgs-review-gha/actions/runs/25237193874


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 14 packages built:</summary>
  <ul>
    <li>free5gc-amf</li>
    <li>free5gc-ausf</li>
    <li>free5gc-bsf</li>
    <li>free5gc-chf</li>
    <li>free5gc-n3iwf</li>
    <li>free5gc-nef</li>
    <li>free5gc-nrf</li>
    <li>free5gc-nssf</li>
    <li>free5gc-pcf</li>
    <li>free5gc-smf</li>
    <li>free5gc-tngf</li>
    <li>free5gc-udm</li>
    <li>free5gc-udr</li>
    <li>free5gc-upf</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 14 packages built:</summary>
  <ul>
    <li>free5gc-amf</li>
    <li>free5gc-ausf</li>
    <li>free5gc-bsf</li>
    <li>free5gc-chf</li>
    <li>free5gc-n3iwf</li>
    <li>free5gc-nef</li>
    <li>free5gc-nrf</li>
    <li>free5gc-nssf</li>
    <li>free5gc-pcf</li>
    <li>free5gc-smf</li>
    <li>free5gc-tngf</li>
    <li>free5gc-udm</li>
    <li>free5gc-udr</li>
    <li>free5gc-upf</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 5 packages failed to build:</summary>
  <ul>
    <li>free5gc-amf</li>
    <li>free5gc-bsf</li>
    <li>free5gc-n3iwf</li>
    <li>free5gc-smf</li>
    <li>free5gc-tngf</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 9 packages built:</summary>
  <ul>
    <li>free5gc-ausf</li>
    <li>free5gc-chf</li>
    <li>free5gc-nef</li>
    <li>free5gc-nrf</li>
    <li>free5gc-nssf</li>
    <li>free5gc-pcf</li>
    <li>free5gc-udm</li>
    <li>free5gc-udr</li>
    <li>free5gc-upf</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>free5gc-amf</summary>
<pre>Running phase: unpackPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;unpackPhase&quot; }
unpacking source archive /nix/store/qj3xlacfiijg5bh9j0c2i0l2ygjj953m-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Building subPackage ./cmd
# github.com/free5gc/sctp
vendor/github.com/free5gc/sctp/sctp.go:504:18: assignment mismatch: 4 variables but c.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:845:21: assignment mismatch: 4 variables but c.conn.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:913:67: too many arguments in call to listenSCTPExtConfig
	have (string, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)
vendor/github.com/free5gc/sctp/sctp.go:923:3: too many arguments in call to dialSCTPExtConfig
	have (string, *SCTPAddr, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)</pre>
</details>
<details>
<summary>free5gc-bsf</summary>
<pre>
[GIN-debug] GET    /nbsf-management/v1/      --&gt; github.com/free5gc/bsf/internal/sbi.getManagementRoutes.func1 (3 handlers)
[GIN-debug] POST   /nbsf-management/v1/pcfBindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.CreatePCFBinding (3 handlers)
[GIN-debug] GET    /nbsf-management/v1/pcfBindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.GetPCFBindings (3 handlers)
[GIN-debug] GET    /nbsf-management/v1/pcfBindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.GetIndPCFBinding (3 handlers)
[GIN-debug] PATCH  /nbsf-management/v1/pcfBindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.UpdateIndPCFBinding (3 handlers)
[GIN-debug] DELETE /nbsf-management/v1/pcfBindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.DeleteIndPCFBinding (3 handlers)
[GIN-debug] POST   /nbsf-management/v1/pcf-ue-bindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.CreatePCFforUEBinding (3 handlers)
[GIN-debug] GET    /nbsf-management/v1/pcf-ue-bindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.GetPCFForUeBindings (3 handlers)
[GIN-debug] PATCH  /nbsf-management/v1/pcf-ue-bindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.UpdateIndPCFforUEBinding (3 handlers)
[GIN-debug] DELETE /nbsf-management/v1/pcf-ue-bindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.DeleteIndPCFforUEBinding (3 handlers)
[GIN-debug] POST   /nbsf-management/v1/pcf-mbs-bindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.CreatePCFMbsBinding (3 handlers)
[GIN-debug] GET    /nbsf-management/v1/pcf-mbs-bindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.GetPCFMbsBinding (3 handlers)
[GIN-debug] PATCH  /nbsf-management/v1/pcf-mbs-bindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.ModifyIndPCFMbsBinding (3 handlers)
[GIN-debug] DELETE /nbsf-management/v1/pcf-mbs-bindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.DeleteIndPCFMbsBinding (3 handlers)
[GIN-debug] POST   /nbsf-management/v1/subscriptions --&gt; github.com/free5gc/bsf/internal/sbi/processor.CreateIndividualSubcription (3 handlers)
[GIN-debug] PUT    /nbsf-management/v1/subscriptions/:subId --&gt; github.com/free5gc/bsf/internal/sbi/processor.ReplaceIndividualSubcription (3 handlers)
[GIN-debug] DELETE /nbsf-management/v1/subscriptions/:subId --&gt; github.com/free5gc/bsf/internal/sbi/processor.DeleteIndividualSubcription (3 handlers)
FAIL	github.com/free5gc/bsf/pkg/service	1.248s
FAIL</pre>
</details>
<details>
<summary>free5gc-n3iwf</summary>
<pre>Running phase: unpackPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;unpackPhase&quot; }
unpacking source archive /nix/store/42zpkw9xbkl4sh34kkv4lwxb30yy039f-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Building subPackage ./cmd
# github.com/free5gc/sctp
vendor/github.com/free5gc/sctp/sctp.go:504:18: assignment mismatch: 4 variables but c.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:845:21: assignment mismatch: 4 variables but c.conn.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:913:67: too many arguments in call to listenSCTPExtConfig
	have (string, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)
vendor/github.com/free5gc/sctp/sctp.go:923:3: too many arguments in call to dialSCTPExtConfig
	have (string, *SCTPAddr, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)</pre>
</details>
<details>
<summary>free5gc-smf</summary>
<pre>2026-05-01T23:53:49.321934000Z [WARN][SMF][CTX] NRF Uri is empty! Using localhost as NRF IPv4 address.
2026-05-01T23:53:49.325691000Z [ERRO][SMF][PFCP] Failed to listen: listen udp :8805: bind: operation not permitted
2026-05-01T23:53:49.326395000Z [FATA][SMF][PFCP] panic: runtime error: invalid memory address or nil pointer dereference
goroutine 51 [running]:
runtime/debug.Stack()
	/nix/store/g0b45l45nlb1kqiywzja7fh49yn7lla5-go-1.26.2/share/go/src/runtime/debug/stack.go:26 +0x5e
github.com/free5gc/smf/internal/pfcp/udp.Run.func1()
	/nix/build/nix-6676-1982425305/source/internal/pfcp/udp/udp.go:26 +0x2b
panic({0x101901660?, 0x101abcfb0?})
	/nix/store/g0b45l45nlb1kqiywzja7fh49yn7lla5-go-1.26.2/share/go/src/runtime/panic.go:860 +0x13a
github.com/free5gc/smf/internal/pfcp/udp.Run(0x101a25bc8)
	/nix/build/nix-6676-1982425305/source/internal/pfcp/udp/udp.go:40 +0x151
github.com/free5gc/smf/internal/pfcp/message_test.TestSendHeartbeatResponse(0x1a15805a6d88)
	/nix/build/nix-6676-1982425305/source/internal/pfcp/message/send_test.go:29 +0x76
testing.tRunner(0x1a15805a6d88, 0x101a25b18)
	/nix/store/g0b45l45nlb1kqiywzja7fh49yn7lla5-go-1.26.2/share/go/src/testing/testing.go:2036 +0xea
created by testing.(*T).Run in goroutine 1
	/nix/store/g0b45l45nlb1kqiywzja7fh49yn7lla5-go-1.26.2/share/go/src/testing/testing.go:2101 +0x4c5
FAIL	github.com/free5gc/smf/internal/pfcp/message	1.051s
FAIL</pre>
</details>
<details>
<summary>free5gc-tngf</summary>
<pre>Running phase: unpackPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;unpackPhase&quot; }
unpacking source archive /nix/store/wxgap7h45is6lv6rrhainif534j2ki55-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Building subPackage ./cmd
# github.com/free5gc/sctp
vendor/github.com/free5gc/sctp/sctp.go:504:18: assignment mismatch: 4 variables but c.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:845:21: assignment mismatch: 4 variables but c.conn.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:913:67: too many arguments in call to listenSCTPExtConfig
	have (string, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)
vendor/github.com/free5gc/sctp/sctp.go:923:3: too many arguments in call to dialSCTPExtConfig
	have (string, *SCTPAddr, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 5 packages failed to build:</summary>
  <ul>
    <li>free5gc-amf</li>
    <li>free5gc-bsf</li>
    <li>free5gc-n3iwf</li>
    <li>free5gc-smf</li>
    <li>free5gc-tngf</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 9 packages built:</summary>
  <ul>
    <li>free5gc-ausf</li>
    <li>free5gc-chf</li>
    <li>free5gc-nef</li>
    <li>free5gc-nrf</li>
    <li>free5gc-nssf</li>
    <li>free5gc-pcf</li>
    <li>free5gc-udm</li>
    <li>free5gc-udr</li>
    <li>free5gc-upf</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>free5gc-amf</summary>
<pre>unpacking source archive /nix/store/qj3xlacfiijg5bh9j0c2i0l2ygjj953m-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Building subPackage ./cmd
# github.com/free5gc/sctp
vendor/github.com/free5gc/sctp/sctp.go:504:18: assignment mismatch: 4 variables but c.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:845:21: assignment mismatch: 4 variables but c.conn.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:913:67: too many arguments in call to listenSCTPExtConfig
	have (string, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)
vendor/github.com/free5gc/sctp/sctp.go:923:3: too many arguments in call to dialSCTPExtConfig
	have (string, *SCTPAddr, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)</pre>
</details>
<details>
<summary>free5gc-bsf</summary>
<pre>
[GIN-debug] GET    /nbsf-management/v1/      --&gt; github.com/free5gc/bsf/internal/sbi.getManagementRoutes.func1 (3 handlers)
[GIN-debug] POST   /nbsf-management/v1/pcfBindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.CreatePCFBinding (3 handlers)
[GIN-debug] GET    /nbsf-management/v1/pcfBindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.GetPCFBindings (3 handlers)
[GIN-debug] GET    /nbsf-management/v1/pcfBindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.GetIndPCFBinding (3 handlers)
[GIN-debug] PATCH  /nbsf-management/v1/pcfBindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.UpdateIndPCFBinding (3 handlers)
[GIN-debug] DELETE /nbsf-management/v1/pcfBindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.DeleteIndPCFBinding (3 handlers)
[GIN-debug] POST   /nbsf-management/v1/pcf-ue-bindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.CreatePCFforUEBinding (3 handlers)
[GIN-debug] GET    /nbsf-management/v1/pcf-ue-bindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.GetPCFForUeBindings (3 handlers)
[GIN-debug] PATCH  /nbsf-management/v1/pcf-ue-bindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.UpdateIndPCFforUEBinding (3 handlers)
[GIN-debug] DELETE /nbsf-management/v1/pcf-ue-bindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.DeleteIndPCFforUEBinding (3 handlers)
[GIN-debug] POST   /nbsf-management/v1/pcf-mbs-bindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.CreatePCFMbsBinding (3 handlers)
[GIN-debug] GET    /nbsf-management/v1/pcf-mbs-bindings --&gt; github.com/free5gc/bsf/internal/sbi/processor.GetPCFMbsBinding (3 handlers)
[GIN-debug] PATCH  /nbsf-management/v1/pcf-mbs-bindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.ModifyIndPCFMbsBinding (3 handlers)
[GIN-debug] DELETE /nbsf-management/v1/pcf-mbs-bindings/:bindingId --&gt; github.com/free5gc/bsf/internal/sbi/processor.DeleteIndPCFMbsBinding (3 handlers)
[GIN-debug] POST   /nbsf-management/v1/subscriptions --&gt; github.com/free5gc/bsf/internal/sbi/processor.CreateIndividualSubcription (3 handlers)
[GIN-debug] PUT    /nbsf-management/v1/subscriptions/:subId --&gt; github.com/free5gc/bsf/internal/sbi/processor.ReplaceIndividualSubcription (3 handlers)
[GIN-debug] DELETE /nbsf-management/v1/subscriptions/:subId --&gt; github.com/free5gc/bsf/internal/sbi/processor.DeleteIndividualSubcription (3 handlers)
FAIL	github.com/free5gc/bsf/pkg/service	0.037s
FAIL</pre>
</details>
<details>
<summary>free5gc-n3iwf</summary>
<pre>unpacking source archive /nix/store/42zpkw9xbkl4sh34kkv4lwxb30yy039f-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Building subPackage ./cmd
# github.com/free5gc/sctp
vendor/github.com/free5gc/sctp/sctp.go:504:18: assignment mismatch: 4 variables but c.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:845:21: assignment mismatch: 4 variables but c.conn.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:913:67: too many arguments in call to listenSCTPExtConfig
	have (string, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)
vendor/github.com/free5gc/sctp/sctp.go:923:3: too many arguments in call to dialSCTPExtConfig
	have (string, *SCTPAddr, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)</pre>
</details>
<details>
<summary>free5gc-smf</summary>
<pre>2026-05-01T23:30:50.541459000Z [WARN][SMF][CTX] NRF Uri is empty! Using localhost as NRF IPv4 address.
2026-05-01T23:30:50.541672000Z [ERRO][SMF][PFCP] Failed to listen: listen udp :8805: bind: operation not permitted
2026-05-01T23:30:50.541851000Z [FATA][SMF][PFCP] panic: runtime error: invalid memory address or nil pointer dereference
goroutine 57 [running]:
runtime/debug.Stack()
	/nix/store/fkzywh7rg8lg1n2p0r161iqjwhz540wx-go-1.26.2/share/go/src/runtime/debug/stack.go:26 +0x64
github.com/free5gc/smf/internal/pfcp/udp.Run.func1()
	/nix/build/nix-10650-3659169903/source/internal/pfcp/udp/udp.go:26 +0x2c
panic({0x1018e5380?, 0x101aa1ff0?})
	/nix/store/fkzywh7rg8lg1n2p0r161iqjwhz540wx-go-1.26.2/share/go/src/runtime/panic.go:860 +0x12c
github.com/free5gc/smf/internal/pfcp/udp.Run(0x101a09748)
	/nix/build/nix-10650-3659169903/source/internal/pfcp/udp/udp.go:40 +0x120
github.com/free5gc/smf/internal/pfcp/message_test.TestSendHeartbeatResponse(0x33414d441448)
	/nix/build/nix-10650-3659169903/source/internal/pfcp/message/send_test.go:29 +0x74
testing.tRunner(0x33414d441448, 0x101a09698)
	/nix/store/fkzywh7rg8lg1n2p0r161iqjwhz540wx-go-1.26.2/share/go/src/testing/testing.go:2036 +0xc4
created by testing.(*T).Run in goroutine 1
	/nix/store/fkzywh7rg8lg1n2p0r161iqjwhz540wx-go-1.26.2/share/go/src/testing/testing.go:2101 +0x3a8
FAIL	github.com/free5gc/smf/internal/pfcp/message	0.026s
FAIL</pre>
</details>
<details>
<summary>free5gc-tngf</summary>
<pre>unpacking source archive /nix/store/wxgap7h45is6lv6rrhainif534j2ki55-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Building subPackage ./cmd
# github.com/free5gc/sctp
vendor/github.com/free5gc/sctp/sctp.go:504:18: assignment mismatch: 4 variables but c.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:845:21: assignment mismatch: 4 variables but c.conn.SCTPRead returns 3 values
vendor/github.com/free5gc/sctp/sctp.go:913:67: too many arguments in call to listenSCTPExtConfig
	have (string, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)
vendor/github.com/free5gc/sctp/sctp.go:923:3: too many arguments in call to dialSCTPExtConfig
	have (string, *SCTPAddr, *SCTPAddr, InitMsg, *RtoInfo, *AssocInfo, int, func(network string, address string, c syscall.RawConn) error)
	want (string, *SCTPAddr, *SCTPAddr, InitMsg, func(network string, address string, c syscall.RawConn) error)</pre>
</details>
</details>

--- a/pkgs/by-name/fr/free5gc-amf/package.nix
+++ b/pkgs/by-name/fr/free5gc-amf/package.nix
@@ -1,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule (finalAttrs: {
+  pname = "free5gc-amf";
+  version = "1.4.3";
+
+  src = fetchFromGitHub {
+    owner = "free5gc";
+    repo = "amf";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-oH01ySslrVhreFIYcEGs3jBZy37wylwhtz8f4VjSlgk=";
+  };
+
+  vendorHash = "sha256-plUoCtHK8/cuYmPosV7ISIlqZGJZUJ4L0Fd0cWV+3zo=";
+
+  ldflags = [
+    "-X github.com/free5gc/util/version.VERSION=v${finalAttrs.version}"
+  ];
+
+  __structuredAttrs = true;
+
+  meta = {
+    description = "Open source 5G core network based on 3GPP R15";
+    homepage = "https://free5gc.org/";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ felbinger ];
+    platforms = lib.platforms.linux;
+  };
+})

--- a/pkgs/by-name/fr/free5gc-ausf/package.nix
+++ b/pkgs/by-name/fr/free5gc-ausf/package.nix
@@ -1,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule (finalAttrs: {
+  pname = "free5gc-ausf";
+  version = "1.4.3";
+
+  src = fetchFromGitHub {
+    owner = "free5gc";
+    repo = "ausf";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-S96QhcerwA2c/fW7J0C4HlcQVeSHrob4VCKOSd4iUpo=";
+  };
+
+  vendorHash = "sha256-8oATdtM8pA8KKgPHFCqsEM0Lo93ZBkoTqLRPfHywCOY=";
+
+  ldflags = [
+    "-X github.com/free5gc/util/version.VERSION=v${finalAttrs.version}"
+  ];
+
+  __structuredAttrs = true;
+
+  meta = {
+    description = "Open source 5G core network based on 3GPP R15";
+    homepage = "https://free5gc.org/";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ felbinger ];
+    platforms = lib.platforms.linux;
+  };
+})

--- a/pkgs/by-name/fr/free5gc-bsf/package.nix
+++ b/pkgs/by-name/fr/free5gc-bsf/package.nix
@@ -1,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule (finalAttrs: {
+  pname = "free5gc-bsf";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "free5gc";
+    repo = "bsf";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-LIsLaAt3EZMUJBIVErFUP/DLrJnFhu+8WJMiKAgxUYE=";
+  };
+
+  vendorHash = "sha256-2T6Al3/Z7TCRIAk96ygT9LOe2hmqsyPfKuXqjW8LGig=";
+
+  ldflags = [
+    "-X github.com/free5gc/util/version.VERSION=v${finalAttrs.version}"
+  ];
+
+  __structuredAttrs = true;
+
+  meta = {
+    description = "Open source 5G core network based on 3GPP R15";
+    homepage = "https://free5gc.org/";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ felbinger ];
+    platforms = lib.platforms.linux;
+  };
+})

